packages/appeals-service-api/src/controllers/responses.js

```diff
@@ -15,4 +15,18 @@ const patchResponseByReferenceId = async (req, res) => {
 	}
 };
 
-module.exports = { patchResponseByReferenceId };
+const getResponseByReferenceId = async (req, res) => {
+	let statusCode = 200;
+	let body = {};
+	try {
+		body = await getResponse(req.params.formId, req.params.referenceId, req.params?.projection);
+	} catch (error) {
+		logger.error(`Failed to patch response: ${error.code} // ${error.message.errors}`);
```

Comment:
Shouldn't this say `Failed to get response` rather than `Failed to patch response`?